README

```diff
@@ -1327,23 +1330,69 @@ of the nullmodem connection. These are all parameters:
 Example: Be a server listening on TCP port 5000.
    serial1=nullmodem server:<IP or name of the server> port:5000 rxdelay:1000
 
-BBS Gaming
-----------
-DOSBox's serial interface can emulate a telephone modem, which allows original
-DOS terminal applications to either dial or host a BBS on the Internet 
-via the Telnet protocol.
 
-First, configure DOSBox with a serial port emulating a modem:
+BBS Support: softmodem connection over TCP
+------------------------------------------
+DOSBox emulates a hardware modem allowing one to either host or dial a BBS
+across the Internet.
+
+To serve a BBS behind a Telnet interface:
+
+ 1) Configure DOSBox's serial port as follows:
+
+    [serial]
+    serial1 = modem telnet:1 listenport:2323
+
+    A port greater than 1024 allows you to run DOSBox as a normal user instead
+    of requiring root or administrator rights.
+
+ 2) Configure your machine to use a static IP address or be assigned a static
+    IP via DHCP, which can typically be configured in your router.
+
+ 3) If your machine is behind a router/firewall, add a port-fowarding entry
+    to listen on TCP port 23 and pass it through to port 2323 to your machine's
+    IP address.  This allows Internet users to "dial" your BBS using the default
+    Telnet port.
+
+To dial a BBS:
+
+ 1) Configure DOSBox's serial port as follows:
+
+    [serial]
+    serial1 = modem listenport:2323
 
-   [serial]
-   serial1 = modem listenport:2323
+    If you're positive that all the BBSes you plan to dial are hosted
+    behind Telnet servers, then enable telnet-mode as shown in step
+    1) in the BBS server section above (add telnet:1).
```

Comment:
Yes, "in the BBS server section", which is the section above this client section :-)

But it will be clearer if I just repeat the content and drop the reference.

Reply:
Got it :-)